src/Configuration.ts

```diff
@@ -285,8 +285,20 @@ export const load = (args: {
       )
     )
 
+    const srcDir = Option.match(config, {
+      onNone: () => args.srcDir,
+      onSome: ({ srcDir }) => srcDir || args.srcDir,
+    })
+
+    const outDir = Option.match(config, {
+      onNone: () => args.outDir,
+      onSome: ({ outDir }) => outDir || args.outDir,
+    })
```

Comment:
```suggestion
    const srcDir = config.pipe(
      Option.flatMapNullable((config) => config.srcDir),
      Option.getOrElse(() => args.srcDir)
    )

    const outDir = config.pipe(
      Option.flatMapNullable((config) => config.outDir),
      Option.getOrElse(() => args.outDir)
    )
```

Reply:
Yes, your solution is more elegant.

Do you want me to modify my PR?

Reply:
(FYI if you agree with the changes you can simply click the "Commit suggestion" button)

Reply:
Sorry I don t have access to my PC tonight. And no such button on the phone github version. Will do it tomorrow morning.

Thanks and good night